src/tools/CrosshairsTool.test.js

```diff
@@ -1,6 +1,6 @@
 import CrosshairsTool from './CrosshairsTool.js';
 
-jest.mock('../import.js', () => ({
+jest.mock('../importInternalModule.js', () => ({
```

Comment:
I wonder why we're mocking this.

Reply:
No idea actually, I just changed the name but good this is flagged, this is in a lot of places and isn't used, i'll clean it up.